pom.xml

```diff
@@ -54,7 +54,7 @@
         <accumulo.version>1.7.2</accumulo.version>
         <avro.version>1.7.7</avro.version>
         <hadoop.version>2.6.5</hadoop.version>
-        <jackson.version>2.6.5</jackson.version>
+        <jackson.version>2.3.1</jackson.version>
```

Comment:
Using this version is now not compatible with the version of scala/spark - this exception is thrown:
com.fasterxml.jackson.databind.JsonMappingException: Jackson version is too old 2.3.1
	at com.fasterxml.jackson.module.scala.JacksonModule$class.setupModule(JacksonModule.scala:56)
	at com.fasterxml.jackson.module.scala.DefaultScalaModule.setupModule(DefaultScalaModule.scala:19)
	at com.fasterxml.jackson.databind.ObjectMapper.registerModule(ObjectMapper.java:537)
	at org.apache.spark.rdd.RDDOperationScope$.<init>(RDDOperationScope.scala:82)
	at org.apache.spark.rdd.RDDOperationScope$.<clinit>(RDDOperationScope.scala)
	... 37 more